app/(admin)/admin/notifications/actions/sendEmailNotification.test.tsx

```diff
@@ -0,0 +1,35 @@
+'use server';
+import { messaging } from "@/api/serverConfig";
+import { sendEmailNotifications } from "./sendEmailNotification";
+
+jest.mock('./sendEmailNotification', () => ({
+    sendEmailNotifications: jest.fn(),
+}))
+
+jest.mock('@/api/serverConfig', () => ({
+    messaging: {
+        createEmail: jest.fn(),
+    },
+}));
+
+describe('SendEmailNotification', () => {
+    it('should send email with provided information', async () => {
+        const content = 'Test';
+        const groupEmailTest = ['123456', '12345', '1234'];
```

Comment:
I wonder if this mock data could be mock emails instead to be more accurate to the actual data that will be used

Reply:
I can't mock emails because the data that goes through is userID's. So in the test, I just use random numbers to act like userID's

Reply:
I would change the variable name in that case, because this variable name suggests these should be emails.